dts/arm/nordic/nrf54h20_cpurad.dtsi

```diff
@@ -59,21 +59,37 @@ wdt011: &cpurad_wdt011 {};
 };
 
 &dppic130 {
-	owned-channels = <0>;
-	sink-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <0 2>;
+	source-channels = <3>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &dppic132 {
-	owned-channels = <0>;
-	source-channels = <0>;
-	nonsecure-channels = <0>;
+	owned-channels = <0 2 3>;
+	sink-channels = <3>;
+	source-channels = <0 2>;
+	nonsecure-channels = <0 2 3>;
 	status = "okay";
 };
 
 &ipct130 {
-	owned-channels = <0>;
-	source-channel-links = <0 3 0>;
 	status = "okay";
+	owned-channels = <0 2>;
+	sink-channel-links = <0 3 0>;
+	source-channel-links = <0 3 0
+					2 3 2>;
+};
+
+&cpurad_ipct {
+	status = "okay";
+	sink-channel-links = <2 13 2>;
+	source-channel-links = <0 13 0>;
+};
+
+&dppic020 {
+	status = "okay";
+	sink-channels = <15>;
+	source-channels = <11>;
```

Comment:
I think those values can be removed, because the the nrf-regtool does not read those settings and the radio core firmware can configure them in any way.

Reply:
Ok